firewall-custom/99firewallcustom.init.d

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh /etc/rc.common
+# Copyright (C) 2022 Minim Inc
+
+START=99
+STOP=99
+
+boot() {
+    # disable SIP nat module
+    sed -i -e 's/nf_nat_sip//g' /etc/modules.d/nf-nathelper-extra
```

Comment:
This will only happen after modules are already loaded, so it won't take effect until the next boot, right?

Reply:
Yes, that was the intention since it's not a mission critical change.  Should I add an rmmod perhaps?

Reply:
yea, I don't think its critical.  But now that you mention it, I guess an rmmod wouldn't hurt either